phpBB/includes/functions_admin.php

```diff
@@ -698,6 +698,28 @@ function move_posts($post_ids, $topic_id, $auto_sync = true)
 		sync('topic_attachment', 'topic_id', $topic_ids);
 		sync('topic', 'topic_id', $topic_ids, true);
 		sync('forum', 'forum_id', $forum_ids, true, true);
+
+		/**
+		 * Perform additional actions after move post sync
+		 *
+		 * @event core.move_posts_sync_after
+		 * @var	array	post_ids	Array of the moved post ids
+		 * @var	string	topic_id	The topic id the posts are moved to
```

Comment:
This was copied from one of the two move_posts events above, so I've fixed those too.